bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Navigation.java

```diff
@@ -42,9 +42,19 @@
      * omitted, {@code false} otherwise).
      *
      * @since com.adobe.cq.wcm.core.components.models 12.2.0
+     * @deprecated since 12.11.0
      */
+    @Deprecated
     String PN_SKIP_NAVIGATION_ROOT = "skipNavigationRoot";
 
+    /**
+     * Name of the resource / configuration policy property that defines the start level of navigation, relative to the
+     * {@link #PN_NAVIGATION_ROOT}, from which to start building the tree. The property should provide a Long value.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.11.0
+     */
+    String PN_ROOT_LEVEL = "rootLevel";
```

Comment:
I'd rename as `PN_STRUCTURE_START = "structureStart"` to be consistent with the existing `structureDepth` config which is also relative to the navigation root.

Reply:
Done in https://github.com/adobe/aem-core-wcm-components/pull/836/commits/a051a2f1675345cb3ac0ae1ae4d6656170c1d947